clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/servergroup/ServerGroupParameters.groovy

```diff
@@ -80,6 +81,10 @@ class ServerGroupParameters {
       user_data            : rawUserData ?: null,
       resource_filename    : resourceFilename ?: ServerGroupConstants.SUBTEMPLATE_FILE
     ]
+    if (floatingNetworkId) {
+      params << [floating_network_id: floatingNetworkId]
```

Comment:
I found that if the key is present, openstack rejects the stack creation due to an unexpected (even if null) parameter.